Update/omake_01.txt

```diff
@@ -1594,6 +1488,8 @@ void main()
 		   NULL, " So half of us think it was a curse, and half of us think it was people?\"", GetGlobalFlag(GLinemodeSp));
 	if (GetGlobalFlag(GADVMode)) { ClearMessage(); } else { OutputLineAll(NULL, "\n", Line_ContinueAfterTyping); }
 
+	if(GetGlobalFlag(GCensor) == 0){ModCallScriptSection("zomake_01_vm00_n01","dialog004");}
```

Comment:
You sure about this? We generally use the GCensor for places where PS3 and original have different dialogue. But I don't think we ever used it to simply cut out something.

Reply:
I think I was being too sensitive to the idea of having unvoiced lines here. I don't mind putting it back.